catalog/core/catalog-core-impl/pubsub/src/main/java/ddf/catalog/pubsub/criteria/contextual/CaseSensitiveContextualAnalyzer.java

```diff
@@ -68,9 +68,7 @@
      * Specifies whether deprecated acronyms should be replaced with HOST type. See
      * {@linkplain https://issues.apache.org/jira/browse/LUCENE-1068}
      */
-    private final boolean replaceInvalidAcronym, enableStopPositionIncrements;
-
-    private final Version matchVersion;
```

Comment:
❓ I'm kind of worried about why we don't need these... while I can't find a specific place where they're used it seems like we need them if we were taking them as constructor args.